pkg/kv/kvserver/replica_raft.go

```diff
@@ -1092,9 +1107,25 @@ func (r *Replica) refreshProposalsLocked(
 		log.Fatalf(ctx, "refreshAtDelta specified for reason %s != reasonTicks", reason)
 	}
 
+	var maxInflightDurationReq *roachpb.BatchRequest
+	var maxInflightDuration time.Duration
```

Comment:
nit: consider calling these e.g. `maxSlowProposalDuration` or something, since they're conditional on the proposals being considered slow.